.changelog/12536.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+server: Add the `connect.enable_serverless_plugin` configuration option.
```

Comment:
So sometimes for new features that require a lot of changes to make function we ignore line item changes like this that all contribute to "add the big new splashy thing", so that there's just one big "make new thing" line item in the changelog at release time.